pkg/devfile/adapters/common/command.go

```diff
@@ -133,9 +143,14 @@ func validateCommandsForGroup(data data.DevfileData, groupType common.DevfileCom
 // 4. command must have group
 func validateCommand(data data.DevfileData, command common.DevfileCommand) (err error) {
 
-	// type must be exec
-	if command.Exec == nil {
-		return fmt.Errorf("command must be of type \"exec\"")
+	// type must be exec or composite
+	if command.Exec == nil && command.Composite == nil {
```

Comment:
Pls update the function description to include composite command as well

Reply:
Can we add some comp cmd unit test cases for this func?

Reply:
Done